client/src/index.js

```diff
@@ -1,15 +1,16 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import { Provider } from 'react-redux';
-import { createStore, applyMiddleware } from 'redux';
-import reducers from './reducers';
+import React from "react";
+import ReactDOM from "react-dom";
+import { Provider } from "react-redux";
+import { createStore, applyMiddleware } from "redux";
+import reducers from "./reducers";
 
-import App from './App';
+import App from "./App";
 
 const store = createStore(reducers, {}, applyMiddleware());
 
 ReactDOM.render(
-<Provider store={store}>
-  <App />
-</Provider>,
-  document.getElementById('root'));
+  <Provider store={store}>
+    <App />
+  </Provider>,
+  document.getElementById("root")
+);
```

Comment:
make sure you have the same prettier settings!!
in this case single qoutes for strings